### DISCUSSION TITLE
Open in Vim(MacVim) and Neovim(Neovide)

### DISCUSSION THREAD

**Discussion by suliveevil** [Taitava/obsidian-shellcommands#271] [Show and tell]
`/opt/homebrew/bin/neovide +{{caret_position:line}} {{file_path:absolute}}`

This can open file in Vim(MacVim) with cursor line, but I didn't find a way to locate cursor column.

https://github.com/vim/vim/issues/11367

**Comment by suliveevil** [comment]
Update:Neovide 0.12.0
Update:Support line and column now:

```shell
/opt/homebrew/bin/mvim {{file_path:absolute}} -c 'normal {{caret_position:line}}G{{caret_position:column}}|'

/opt/homebrew/bin/neovide {{file_path:absolute}} -- -c 'normal {{caret_position:line}}G{{caret_position:column}}|'
```

**Reply by Taitava** [reply]
## Nice, thank you! 🙂
